docs/rfcs/014-delivery-channels-database.md

```diff
@@ -0,0 +1,201 @@
+# Delivery channel database design
+
+## Brief
+
+This RFC is related to the [table design for delivery channels](https://github.com/dlcs/protagonist/issues/618) to help show a table design for the additional tables required for the delivery channels work
+
+## Delivery Channel Policy Table
+
+![delivery channel policies](img/DeliveryChannels.png)

```

Comment:
This image doesn't reflect the below image - it shows `iiif-img` channel having policyData which doesn't match with below table + description